### DISCUSSION TITLE
problemas com responsividade

### DISCUSSION THREAD

**Discussion by guilhermejorge0323** [frontendbr/forum#2490] [Discussão]
Ola estou fazendo alguns projetos em html e css e nao estou conseguindo fazer a responsividade para telas maiores estou usando o vw vh mas acaba que tenho que colocar muitos mideas e muitas variaveis tambem tenho problemas em colocar responsividade em telas na horizontal poderiam me indicar repositorios de landings pages totalmente responsivas para mim analizar o codigo

**Comment by viniciusbavosa** [comment]
tenta utilizar mais porcentagem e max/min-width. Usando porcentagem os filhos se adaptam ao tamanho do elemento pai. Media-query serve para criar experiências diferentes para viewports específicos, não ajuda muito com a responsividade dos elementos

**Comment by nazarepiedady** [comment]
@guilhermejorge0323, recomendo estudar o tema da responsividade com os módulos de disposição de grade (*grid layout*) e disposição de caixa flexível (*flexbox layout*).
